src/App.css

```diff
@@ -1,11 +1,17 @@
+.siteHeader {
+  height: 75px;
+  background-color:#F8333C;
+  background-image: linear-gradient(to bottom right, #F8333C, #FCAB10);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: 2px solid #FCAB10;
+}
+
 .siteTitle {
-  font-size: 20px;
+  font-family: fantasy;
```

Comment:
I did not think this font family is "fantasy" because it looks like "papyrus"...Not sure if that was what you were going for but we could look into another font that is easily recognized by the browser (if this was the case)